mir-module/src/main/resources/META-INF/resources/js/mir/accesskey.js

```diff
@@ -0,0 +1,244 @@
+const RSC_URL = webApplicationBaseURL + "rsc/accesskey/";
+let labels = undefined;
+
+function getParameterByName(name, url = window.location.href) {
+    name = name.replace(/[\[\]]/g, '\\$&');
+    var regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)'),
+    results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, ' '));
+}
+
+function generateKey(plength) {
+	var keylistalpha = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+	var keylistint = "123456789";
+	var keylistspec = "!@#_%$";
+	var temp = '';
+	var len = plength / 2;
+	var len = len - 1;
+	var lenspec = plength - len - len;
+
+	for (i = 0; i < len; i++)
+		temp += keylistalpha.charAt(Math.floor(Math.random() * keylistalpha.length));
+
+	for (i = 0; i < lenspec; i++)
+		temp += keylistspec.charAt(Math.floor(Math.random() * keylistspec.length));
+
+	for (i = 0; i < len; i++)
+		temp += keylistint.charAt(Math.floor(Math.random() * keylistint.length));
+
+	temp = temp.split('').sort(function() {
+		return 0.5 - Math.random()
+	}).join('');
+
+	return temp;
+};
+
+function addAccessKeyTableRow(accessKey) {
+    const table = document.getElementById("accessKeys");
+    const index = table.rows.length++;
+    let row = table.insertRow(-1);
+    row.className = "d-flex";
+    let indexCell = row.insertCell(0);
+    indexCell.className = "th-sm col-auto"
+    indexCell.innerHTML = index;
+    let typeCell = row.insertCell(1);
+    typeCell.className = "th-sm col-3 text-center"
+    typeCell.innerHTML = accessKey["type"];
+    let valueCell = row.insertCell(2);
+    valueCell.className = "th-sm col text-center"
+    valueCell.innerHTML = accessKey["value"];
+
+    var i = document.createElement('i');
+    i.className = "fas fa-edit";
+    var a = document.createElement('a');
+    a.setAttribute("data-toggle", "modal");
+    a.setAttribute("data-id", accessKey["id"]);
+    a.setAttribute("data-type", accessKey["type"]);
+    a.setAttribute("data-value", accessKey["value"]);
+    a.setAttribute("href", "#accessKeyModal");
+    a.className = "editAccessKeyModal"
+    a.appendChild(i);
+    let editCell = row.insertCell(3)
+    editCell.className = "th-sm col-auto";
+    editCell.appendChild(a);
+}
+
+function proccessAccessKeyInformation(accessKeys) {
+    for (key in accessKeys) {
+        if (accessKeys.hasOwnProperty(key)) {
+            const accessKey = accessKeys[key];
+            addAccessKeyTableRow(accessKey);
+        }
+    }
+}
+
+function disableButtons() {
+    $('#accessKeyUpdate').prop("disabled", true);
+    $('#accessKeyDelete').prop("disabled", true);
+    $('#accessKeyNew').prop("disabled", true);
+    $('.closeModal').prop("disabled", true);
+}
+
+function enableButtons() {
+    $('#accessKeyUpdate').prop("disabled", false);
+    $('#accessKeyDelete').prop("disabled", false);
+    $('#accessKeyNew').prop("disabled", false);
+    $('.closeModal').prop("disabled", false);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCoRe-Org/mir/pullRequest?prid=7162253)
```suggestion
    $(".closeModal").prop("disabled", false);
```